### DISCUSSION TITLE
403 Forbidden before index.php

### DISCUSSION THREAD

**Discussion by hobibit** [symfony/symfony#46230] [Q&A]
So I have project in Symfony. Its working fine, but one url doesnt work.
I have "change" public/index.php and add echo 'test'; at the beginning of the file so I see when index.php is executed. And it is executed most time but when I execute one url it doesnet work, doesnt even execute index.php, I just get 403 Forbidden You don't have permission to access this resource.

Why is this happening?

**Comment by alessandro-podo** [comment]
which http-server do you use?

**Reply by hobibit** [reply]
<html><body>


SERVER_SOFTWARE | "Apache"
SERVER_PROTOCOL | "HTTP/1.1"

</body>
</html>

**Reply by hobibit** [reply]
So issue was that text/csv files was allowed but for windows they are application/vnd.ms-excel mime type

https://stackoverflow.com/questions/7076042/what-mime-type-should-i-use-for-csv
